### PR TITLE
Fix mising casadm error message

### PR DESCRIPTION
**Pull request by kmajzero**
This patch fixes missing casadm message when --standby --activate
is used with a direct path to a non-whitelisted device.

Signed-off-by: Krzysztof Majzerowicz-Jaszcz <krzysztof.majzerowicz-jaszcz@intel.com>

### CODE REVIEW

File: casadm/cas_lib.c [hunk 1]
```diff
@@ -3041,6 +3041,8 @@ int standby_activate(int cache_id, const char *cache_device)
 
 	if (set_device_path(cmd.cache_path, sizeof(cmd.cache_path),
```
Comment (arutk) [inline casadm/cas_lib.c:3042]: I would move this message inside set_device_path() - it is now printed almost everywhere whenever set_device_path() is printed + there are cases where set_device_path() prints other messages and it would be good to have just one.

File: casadm/cas_lib.c [hunk 2]
```diff
@@ -1901,15 +1899,11 @@ int core_pool_remove(const char *core_device)
...
 			    core_device, MAX_STR_LEN) != SUCCESS) {
-		cas_printf(LOG_ERR, "Please use correct by-id path to the device "
-			   "%s.\n", core_device);
 		close(fd);
 		return FAILURE;
 	}
 
 	if (ioctl(fd, KCAS_IOCTL_CORE_POOL_REMOVE, &cmd) < 0) {
-		cas_printf(LOG_ERR, "Error while removing device %s from core pool\n",
-				core_device);
```
Comment (arutk) [inline casadm/cas_lib.c:1912]: this one should stay

File: casadm/cas_lib.c [hunk 3]
```diff
@@ -3040,9 +3033,8 @@ int standby_activate(int cache_id, const char *cache_device)
 	};
 
 	if (set_device_path(cmd.cache_path, sizeof(cmd.cache_path),
-			    cache_device, MAX_STR_LEN) != SUCCESS) {
+			    cache_device, MAX_STR_LEN) != SUCCESS)
 		return FAILURE;
-	}
```
Comment (robertbaldyga) [inline casadm/cas_lib.c:3045]: Is there any reason why these brackets disappeared?

File: casadm/cas_lib.c [hunk 4]
```diff
@@ -1901,15 +1899,13 @@ int core_pool_remove(const char *core_device)
...
-		cas_printf(LOG_ERR, "Please use correct by-id path to the device "
-			   "%s.\n", core_device);
 		close(fd);
 		return FAILURE;
 	}
 
 	if (ioctl(fd, KCAS_IOCTL_CORE_POOL_REMOVE, &cmd) < 0) {
 		cas_printf(LOG_ERR, "Error while removing device %s from core pool\n",
-				core_device);
+			   core_device);
```
Comment (robertbaldyga) [inline casadm/cas_lib.c:1908]: This change seems to be unrelated (and inconsistent with our coding style).

**Review by robertbaldyga — CHANGES_REQUESTED** [review]

**Review by mmichal10 — APPROVED** [review]

**Review by jfckm — APPROVED** [review]
